sbatch_wrapper/__init__.py

```diff
@@ -47,7 +47,7 @@ def debug(*args, **kwargs):
                     continue
                 if "--exclusive" in l:
                     debug("found exclusive")
-                    used_exclusive = True
+                    used_exclusive = used_exclusive
```

Comment:
That is incorrect, test are failing. You wan to change `used_exclusive` on line 92, not here.